routing/car_router.cpp

```diff
@@ -578,6 +581,28 @@ bool CarRouter::DoesEdgeIndexExist(Index::MwmId const & mwmId)
   return true;
 }
 
+bool CarRouter::AllMwmsHaveRoutingIndex() const
+{
+  // The new routing section with IndexGraph was added in december 2016.
+  // If some mwm does not have this section, old cross mwm routing is used.
+  int64_t constexpr kFirstVersionWithRoutingIndex = 161206;
```

Comment:
We have platform/mwm_traits for such things. Please, create a corresponding method in MwmTraits.